redhat-operators/amq-streams/amq-streams.v1.1.0.clusterserviceversion.yaml

```
@@ -0,0 +1,618 @@
+                - name: STRIMZI_DEFAULT_TOPIC_OPERATOR_IMAGE
+                  value: registry.access.redhat.com/amq7/amq-streams-topic-operator:1.1.0
+                - name: STRIMZI_DEFAULT_USER_OPERATOR_IMAGE
+                  value: registry.access.redhat.com/amq7/amq-streams-user-operator:1.1.0
+                - name: STRIMZI_DEFAULT_KAFKA_INIT_IMAGE
+                  value: registry.access.redhat.com/amq7/amq-streams-kafka-init:1.1.0
+                - name: STRIMZI_DEFAULT_TLS_SIDECAR_ZOOKEEPER_IMAGE
+                  value: registry.access.redhat.com/amq7/amq-streams-zookeeper-stunnel:1.1.0
+                - name: STRIMZI_DEFAULT_TLS_SIDECAR_KAFKA_IMAGE
+                  value: registry.access.redhat.com/amq7/amq-streams-kafka-stunnel:1.1.0
+                - name: STRIMZI_DEFAULT_TLS_SIDECAR_ENTITY_OPERATOR_IMAGE
+                  value: registry.access.redhat.com/amq7/amq-streams-entity-operator-stunnel:1.1.0
+                - name: STRIMZI_LOG_LEVEL
+                  value: INFO
+                - name: STRIMZI_CREATE_CLUSTER_ROLES
+                  value: "TRUE"
+                livenessProbe:
+                  httpGet:
+                    path: /healthy
+                    port: 8080
+                  initialDelaySeconds: 10
+                  periodSeconds: 30
+                readinessProbe:
+                  httpGet:
+                    path: /ready
+                    port: 8080
+                  initialDelaySeconds: 10
+                  periodSeconds: 30
+                resources:
+                  limits:
+                    cpu: 1000m
+                    memory: 256Mi
+                  requests:
+                    cpu: 200m
+                    memory: 256Mi
+          strategy:
+            type: Recreate
+  customresourcedefinitions:
+    owned:
+    - name: kafkas.kafka.strimzi.io
+      version: v1alpha1
+      kind: Kafka
+      displayName: Kafka
+      description: Represents a Kafka cluster
+      specDescriptors:
+        - description: The desired number of Kafka brokers.
+          displayName: Kafka Brokers
+          path: kafka.replicas
+          x-descriptors:
+            - 'urn:alm:descriptor:com.tectonic.ui:label'
+        - description: The type of storage used by Kafka brokers
+          displayName: Kafka storage
+          path: kafka.storage.type
+          x-descriptors:
+            - 'urn:alm:descriptor:com.tectonic.ui:label'
+        - description: Limits describes the minimum/maximum amount of compute resources required/allowed
+          displayName: Kafka Resource Requirements
+          path: kafka.resources
+          x-descriptors:
+            - 'urn:alm:descriptor:com.tectonic.ui:resourceRequirements'
+        - description: The desired number of Zookeeper nodes.
+          displayName: Zookeeper Nodes
+          path: zookeeper.replicas
+          x-descriptors:
+            - 'urn:alm:descriptor:com.tectonic.ui:label'
```

Same here - the Operator reconciles updates to this field - why not expose it in the UI as an input?

```suggestion
            - 'urn:alm:descriptor:com.tectonic.ui:podCount'
```